libs/core/src/components/pds-radio/pds-radio.tsx

```
@@ -107,43 +132,59 @@ export class PdsRadio {
   }
 
   render() {
+    const renderLabelAndMessages = () => [
+      <label htmlFor={this.componentId}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)